src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -98,11 +99,15 @@ export class WorkItemDetailComponent implements OnInit {
             .getUser()
             .then(user => this.setLoggedInUser(user));
         }
+        //set the creator/reporterUser         
+        this.reporterUser = this.getReporter(
+          this.workItem.fields['system.creator']);                 
```

Comment:
This is the wrong API and will be removed in https://github.com/almighty/almighty-core/issues/536